code/modules/food&drinks/drinks/drinks/drinkingglass.dm

```diff
@@ -162,6 +162,7 @@
 				icon_state = "tequilasunriseglass"
 				name = "tequila Sunrise"
 				desc = "Oh great, now you feel nostalgic about sunrises back on Terra..."
+			// Toxins Special is also in drinks.dm because shot glasses, mirror any changes there. //
```

Comment:
I'll need to make a changelog for it, but alright. I'll keep an eye out and see if anyone adminhelps about their drinks not working.